File: BubbleSort.py
def BubbleSort(l):
    length=len(l)
    for a in range(length):
        for i in range(length-1-a):
           if l[i]>l[i+1]:
                l[i],l[i+1]=l[i+1],l[i]
    return l
l=[2,1,5,6,2,3]
print(BubbleSort(l))

File: Js/BubbleSort.js
const BubbleSort = (l) => {
  const len = l.length;
  let sortedL = [...l];
  for (let i = 0; i < len - 1; i++) {
    for (let j = i; j < len; j++) {
      if (sortedL[j - 1] > sortedL[j]) {
        [sortedL[j - 1], sortedL[j]] = [sortedL[j], sortedL[j - 1]];
      }
    }
  }
  return sortedL;
};

const l = [2, 5, 1, 7, 3, 9, 43, 12, 13, 13, 54, 23, 11];
console.log(BubbleSort(l));
